lib/jsdom/living/window/Screen-impl.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+// https://drafts.csswg.org/cssom-view-1/#the-screen-interface
+exports.implementation = class ScreenImpl {
+  get availWidth() {
+    return 0;
```

Comment:
These will never get changed, so I'd rather just do something like https://github.com/TimothyGu/jsdom/blob/window-webidl2js/lib/jsdom/living/window/Screen-impl.js

```js
class ScreenImpl {}
ScreenImpl.prototype.availWidth = 0;
ScreenImpl.prototype.availHeight = 0;
ScreenImpl.prototype.width = 0;
ScreenImpl.prototype.height = 0;
ScreenImpl.prototype.colorDepth = 0;
ScreenImpl.prototype.pixelDepth = 0;
```

Reply:
Oh, I wasn't aware of how far you've gotten on the new implementation of window. The refactored version looks great!

Regarding this file in particular, I wouldn't mind making this change. I usually prioritise being consistent across files, but the advantages of making an exception here are pretty clear.

Reply:
I'd also prefer the prototype version for static values (wherever possible).

Reply:
I kind of like the consistency, given that these aren't going to be commonly accessed. But either way.